python-usfm-parser/exp-parser.py

```diff
@@ -0,0 +1,51 @@
+''' Uses the tree-sitter-usfm grammar and converts the parse tree to json'''
+import argparse
+import json
+from tree_sitter import Language, Parser
+
+Language.build_library(
+  # Store the library in the `build` directory
+  'build/my-languages.so',
+
+  # Include one or more languages
+  [
+    '../tree-sitter-exp',
+    # '../tree-sitter-usfm'
+  ]
+)
+
+USFM_LANGUAGE = Language('build/my-languages.so', 'exp')
+parser = Parser()
+parser.set_language(USFM_LANGUAGE)
+
+def parse_n_convert(usfm_string):
+	'''try parsing and on successfull parsing convert parse tree to JSON'''
+	tree = parser.parse(bytes(usfm_string, "utf8"))
+	root_node = tree.root_node
+	# print(root_node.sexp())
+	json_output = {}
+	# filenode = root_node.children[0]
+	idnode = root_node.children[0]
+	assert idnode.type=='idline'
```

Comment:
This maybe because it is still early but just wanted to point out that we shouldn't use `assert` in production code and rather throw `Exception` (sub-classes ideally) along with `try`-`except` statements.